client/src/extension.ts

```diff
@@ -115,3 +122,14 @@ export function deactivate(): Thenable<void> {
     Configuration.dispose();
     return client.stop();
 }
+
+async function ensureDefaultJarInstalled(outputChannel: OutputChannel, version: string = '8.0'): Promise<void> {
+    const extPath: string = extensions.getExtension('shengchen.vscode-checkstyle').extensionPath;
```

Comment:
extensionPath should be able to got from context?

Reply:
Emmmm, I want to get the extension installation path here. Does context have the API for this?

Reply:
After investigation, I find it can be get from context.